### PR TITLE
Add config enabled for disable-able sources

### PR DESCRIPTION
**Pull request by epapineau**
Pull Request
**Are you a current Fivetran customer?** 

Elize Papineau, Senior Analytics Engineer, dbt Labs
Yes, working on behalf of a customer using Fivetran 
**What change(s) does this PR introduce?** 

Passes the configurable variables to `config: enabled` to prevent running source freshness on disabled models
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)
It extends use of existing variables
**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, [Issue](https://github.com/fivetran/dbt_fivetran_log/issues/57)
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [x] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:mask:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/staging/src_fivetran_log.yml [hunk 1]
```diff
@@ -135,6 +135,8 @@ sources:
       - name: account_membership
         identifier: "{{ var('fivetran_log_account_membership_identifier', 'account_membership')}}"
         description: Table of the users given access to this account.
+        meta:
```
Comment (fivetran-joemarkiewicz) [inline models/staging/src_fivetran_log.yml:138]: @fivetran-jamie actually did some digging and found that dbt-core now has a [feature built in](https://docs.getdbt.com/reference/source-configs) to allow for the dynamic disabling of source! 

This means we don't need to use our override of the freshness macro and instead just disable the source freshness altogether! I just tested the new config and it seems to work as intended. 

Would you be able to try on your end the following update for all of the sources you modified to see if it works on your end as well?
```suggestion
        config:
```

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@epapineau thanks so much for making these changes! They look great on my end, but I actually just discovered that there is a [native way](https://docs.getdbt.com/reference/source-configs) we can disable these sources without needing to use the `meta` config and override the freshness macro!

The best option would be to disable the source entirely instead of the `meta` config. Thankfully, the only change needed in your PR is to swap `meta` for `config`. Would you be able to make those changes and see if they work as intended on your end?

File: models/staging/src_fivetran_log.yml (hunk 1 above)
Comment (epapineau) [reply models/staging/src_fivetran_log.yml]: @fivetran-joemarkiewicz requested changes have been made 😎

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@epapineau thanks so much for making the changes 😄 I made two small changes to bump the version in our integration tests project and also made a small edit to the CHANGELOG.

With that, this PR is good to go! Thanks so much for your collaboration and contribution to the package 🙌 These packages are made better by contributors like yourself adding to the code base. I will merge this and plan to cut a release later this morning.

**Pull request merged**